sei/chain.json

```diff
@@ -519,12 +519,45 @@
           "repo": "https://github.com/sei-ibc-go/v3",
           "version": "v3.3.2"
         }
+      },
+      {
+        "name": "v5.9.0",
+        "proposal": 85,
+        "height": 107000672,
+        "recommended_version": "v5.9.0",
+        "compatible_versions": [
+          "v5.9.0"
+        ],
+        "cosmos_sdk_version": "sei-cosmos v0.3.37-0.20240923023912-aa7a702d42cc",
```

Comment:
I could be wrong but I think the check is failing because `sei-cosos v0.3.37-0.20.....` doesn't match `v0.3.3-0.20....`. @JeremyParish69 is my thinking on this correct? @dreamloss try to fix and re-run and see if that makes the check pass. The same might be true for wasm?

Reply:
sei go.mod for v5.9.0 lists `sei-cosmos v0.3.37-0.20240923023912-aa7a702d42cc`, do you mean to change that to v0.3.3?

Reply:
I mean to change `sei-cosmos v0.3.37-0...` to `v0.3.37-0.20240923023912-aa7a702d42cc` and see if the check passes. Same logic for wasmd.

Reply:
I think this value is OK, but further down where the replacement 'sdk' property is isn't correct.